### ISSUE TITLE
Improve set up documentation related to AWS

### ISSUE THREAD

**Issue by jalberto** [chaskiq/chaskiq#703]
**Is your feature request related to a problem? Please describe.**
As a person with no experience with AWS terminology or technology, it is hard to follow this https://dev.chaskiq.io/production-configuration/email-configuration

**Describe the solution you'd like**
I will like a more detailed step-by-step guide or, even better, a set of CLI commands to run to set everything

**Comment by jalberto** [comment]
Some examples:

- in doc you have `SES_USER_NAME: ses user name` but once I generate the credentials I have either `IAM username` and `smtp username`
- in doc you have `SES_ADDRESS: email address from amazon ses service` which email address? is the same one as used in other ENVs?
- which permissions needs the IAM user for chaskiq?

I am pretty sure this is obvious to someone already familiar with AWS, but can be confusing for people used to different terminology

**Comment by michelson** [comment]
Hi, you only need the AWS credentials to use SES, the SES_USER_NAME and SES_ADDRESS are not needed. 
I will try to document that article with a more detailed explanation!

The AWS user needs to have allowed permissions to SES, SNS and S3.

**Comment by jalberto** [comment]
Hi @michelson I am still have some issues understanding https://dev.chaskiq.io/production-configuration/email-configuration

- is `sns-bounces-sns-topic` explained in other doc? It seems to be used before it gets defined
- In the sections `Create SNS topics`
  - How many topics do I need to create?
  - Where is ` https://YOURDOMAIN/api/v1/hooks` used?
- in `Attach your SNS to SES configuration set`
  - Which topics should be attached to SES?
